repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java

```diff
@@ -553,6 +560,16 @@ private void addIndexForType(AtlasGraphManagement management, AtlasBaseTypeDef t
             if (CollectionUtils.isNotEmpty(attributeDefs)) {
                 for (AtlasAttributeDef attributeDef : attributeDefs) {
                     createIndexForAttribute(management, structDef, attributeDef);
+                    if(!attributeDef.getIsIndexCreated() && !attributeDef.getIndexPresent()) {
+                        typeDef.setIndexCreated(false);
+                        try {
```

Comment:
Why `try-catch` for throwing exception, and we are eating away the exception too ? What is the end goal for this code block ?